flang/test/Parser/OpenMP/sentinels.f

```diff
@@ -0,0 +1,39 @@
+! RUN: %flang_fc1 -fopenmp -E %s | FileCheck %s
+! CHECK:      program main
+! CHECK:       interface
+! CHECK:        subroutine sub(a)
+! CHECK:!dir$ ignore_tkr a
+! CHECK:          real(4):: a
+! CHECK:        end subroutine
+! CHECK:       end interface
+! CHECK:      PRINT *, "Is '    '"
+! CHECK:  123 PRINT *, "Is '123 '"
+
+!@cuf subroutine atcuf;
+      program main
+       interface
+        subroutine sub(a)
+!dir$ ignore_tkr a
```

Comment:
Can you add a second dummy argument (say `b`) and split an `ignore_tkr` directive like so (mind the extra blank at the beginning of the continuing line), so we have a test for a continuation as well?

```fortran
        subroutine sub(a, b)
!dir$ ignore_tkr
!dir$+ b
!dir$ ignore_tkr a
      real(4):: a, b
```